ZeldaFullEditor/ZSCustomOverworld.asm

```diff
@@ -9,68 +48,256 @@ pushpc
+NewNMITarget2 = $04D9 ; [0x02]
+NewNMISource2 = $04DB ; [0x02]
+NewNMICount2 = $04DD ; [0x02]
+
+; Hooks
+Sound_LoadLightWorldSongBank               = $008913
+GFXSheetPointers_sprite_bank               = $00CFF3
+GFXSheetPointers_sprite_high               = $00D0D2
+GFXSheetPointers_sprite_low                = $00D1B1
+DecompOwAnimatedTiles                      = $00D394
+GetAnimatedSpriteTile                      = $00D4DB
+GetAnimatedSpriteTile_variable             = $00D4ED
+LoadTransAuxGFX_sprite_continue            = $00D706
+PrepTransAuxGFX                            = $00DF1A
+Do3To4High16Bit                            = $00DF4F
+Do3To4Low16Bit                             = $00DFB8
+InitTilesets                               = $00E19B
+CopyFontToVram                             = $00E556
+Decomp_bg_variable                         = $00E78F
+OverworldPalettesScreenToSet               = $00FD1C
+
+Credits_LoadScene_PrepGFX_sprite_gfx       = $0285E2
+Credits_LoadScene_PrepGFX_sprite_palette   = $0285F3
+DeleteCertainAncillaeStopDashing           = $028B0C
+OWOverlay_HShift                           = $02A46D
+OWOverlay_VShift                           = $02A471
+Overworld_FinishTransGfx_firstHalf_Retrun  = $02ABC5
+Overworld_LoadSubscreenAndSilenceSFX1      = $02AF19
+Overworld_ReloadSubscreenOverlayAndAdvance = $02B1F4
+Dungeon_LoadPalettes_cacheSettings         = $02C65F
+SpecialOverworld_CopyPalettesToCache       = $02C6EB
+LoadSubscreenOverlay                       = $02FD0D
+
+Link_ItemReset_FromOverworldThings         = $07B107
+
+Tagalong_Init                              = $099EFC
+Sprite_ReinitWarpVortex                    = $09AF89
+Sprite_ResetAll                            = $09C44E
+Sprite_OverworldReloadAll                  = $09C499
+
+Overworld_SetFixedColorAndScroll           = $0BFE70
+
+Overworld_LoadPalettes                     = $0ED5A8
+Palette_SetOwBgColor_Long                  = $0ED618
+LoadGearPalettes_bunny                     = $0ED6DD
+
+PaletteIDtoOffset_OW2                      = $1BEC3B
+Palette_SpriteAux3                         = $1BEC77
+Palette_MainSpr                            = $1BEC9E
+Palette_SpriteAux1                         = $1BECC5
+Palette_SpriteAux2                         = $1BECE4
+Palette_Sword                              = $1BED03
+Palette_Shield                             = $1BED29
+Palette_MiscSpr                            = $1BED6E
+Palette_ArmorAndGloves                     = $1BEDF9
+Palette_Hud                                = $1BEE52
+Palette_OverworldBgAux3                    = $1BEEA8
+Palette_OverworldBgMain                    = $1BEEC7
+
+; ==============================================================================
+; Debug addresses:
+; ==============================================================================
+
+; These can be used to turn off each hook used. Some are reliant on eachother
+; and disabling only some can break entire features.
```

Comment:
Add a todo to log the dependencies and maybe build that into the naming/labeling structure?

Reply:
Done.